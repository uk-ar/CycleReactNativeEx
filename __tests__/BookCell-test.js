// __tests__/BookCell-test.js
import 'react-native';
import React from 'react';

//import '../auto_mock_off';
//import { BookRow, LibraryStatus,BookCell } from '../BookCell';
//import LibraryStatus from '../BookCell';
// Note: test renderer must be required after react-native.

/* jest
 *   .disableAutomock()
 *   .setMock('Icon', {})*/
/* jest.mock('Icon', () => {
 *   const jestReactNative = require('jest-react-native');
 *   return jestReactNative.mockComponent('Icon');
 * });*/
//jest.mock('Icon')
//jest.enableAutomock() 

//var FAIcon = require('react-native-vector-icons/FontAwesome');
var {BookRow, LibraryStatus,BookCell} = require('../BookCell');

import renderer from 'react-test-renderer';

it('renders correctly', () => {
  const tree = renderer.create(
    <LibraryStatus />
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
