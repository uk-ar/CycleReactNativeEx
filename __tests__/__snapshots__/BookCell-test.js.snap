exports[`test renders correctly 1`] = `
<View
  style={
    Array [
      Object {
        "flexDirection": "row"
      }
    ]
  }>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail">
    蔵書確認中
  </Text>
  <ActivityIndicator
    animating={true}
    color="#999999"
    hidesWhenStopped={true}
    size="small" />
</View>
`;
